addons/website_sale_stock/static/src/js/website_sale.js

```diff
@@ -3,6 +3,7 @@
 import { WebsiteSale } from '@website_sale/js/website_sale';
 import { rpc } from "@web/core/network/rpc";
 import { isEmail } from '@web/core/utils/strings';
+import VariantMixin from "@website_sale/js/sale_variant_mixin";
```

Comment:
seems already import thanks to: `import { WebsiteSale } from '@website_sale/js/website_sale';` 🤔

Reply:
as discussed it is needed as we are calling a method form this object in the end of the file.

Reply:
my bad it is indeed needed